app/lib/features/profile/pages/my_profile_page.dart

```diff
@@ -235,42 +231,66 @@ class MyProfile extends ConsumerWidget {
                           ),
                         ],
                       ),
-                      const SizedBox(height: 30),
-                      OutlinedButton.icon(
-                        icon: const Icon(Atlas.construction_tools_thin),
-                        onPressed: () =>
-                            context.pushNamed(Routes.settings.name),
-                        label: const Text('Settings'),
-                      ),
-                      const SizedBox(height: 10),
-                      OutlinedButton.icon(
-                        icon: const Icon(Atlas.laptop_screen_thin),
-                        onPressed: () =>
-                            context.pushNamed(Routes.settingSessions.name),
-                        label: const Text('Sessions'),
-                      ),
-                      const SizedBox(height: 30),
-                      OutlinedButton.icon(
-                        icon: const Icon(Atlas.exit_thin),
-                        onPressed: () => logoutConfirmationDialog(context, ref),
-                        label: const Text('Logout'),
+                      const SizedBox(height: 5),
+                        Container(
+                        margin: const EdgeInsets.symmetric(vertical: 10),
+                        decoration: BoxDecoration(
+                          color: const Color(0xFF122D46),
```

Comment:
eeehh... let's not hard-code colors. Please use the AppThemes and add a color if necessary.

Reply:
Is there a format used in naming the custom color schemes ?

Reply:
@kokodavid well, being specific I'd say. Like `DangerBoxDecorationColor`, is what I'd propose here.